core/trino-main/src/main/java/io/trino/sql/planner/QueryPlanner.java

```diff
@@ -1778,6 +1781,7 @@ private PlanBuilder planWindow(
                                 return coercions.get(argument).toSymbolReference();
                             })
                             .collect(toImmutableList()),
+                    windowFunction.getOrderBy().map(orderBy -> translateOrderingScheme(orderBy.getSortItems(), coercions::get)),
```

Comment:
instead of .map you should use .ifPresent

Reply:
No. I am translating one Optional to another Optional